packages/cli/src/lib/bundler/paths.ts

```diff
@@ -48,10 +48,15 @@ export function resolveBundlingPaths(options: BundlingPathsOptions) {
     }
   }
 
+  // Backend plugin dev run file
+  const targetRunFile = paths.resolveTarget('src/run.ts');
```

Comment:
i guess it would be even nicer if we used `dev/index.ts` like the frontend things - but we'll leave that for a later PR

Reply:
Yep, @Rugvip had the same idea, but what if `run` is used for prod-standalone deployment?

Reply:
Right but the one that specifically `yarn start` runs, may be different from the one you want in a prod setting right? Maybe we could even have those two be clearly different things. The former may for example use an in-memory database hardcoded (even with some mock data force inserted for easy local running), while the latter probably requires config or env vars and connects to a real database